credstash.py

```diff
@@ -398,11 +421,32 @@ def getSecret(name, version="", region=None,
                 "Item {'name': '%s', 'version': '%s'} couldn't be found." % (name, version))
         material = response["Item"]
 
+    return decrypt(
```

Comment:
The main change in this function is that we call `decrypt` at the end, separating the DynamoDB interaction from the KMS one.